src/main/java/de/tum/in/www1/artemis/repository/ParticipationRepository.java

```diff
@@ -18,8 +18,8 @@
     @Query("select distinct p from Participation p left join fetch p.submissions left join fetch p.results where p.id = :#{#participationId}")
     Participation getOneWithEagerSubmissionsAndResults(@Param("participationId") Long participationId);
 
-    @Query("select p from Participation p left join fetch p.results pr left join fetch pr.feedbacks prf where p.id = :participationId and (pr.id = (select max(id) from p.results) or pr.id = null)")
-    Optional<Participation> findByIdWithLatestResultAndFeedbacks(@Param("participationId") Long participationId);
+    @Query("select p from Participation p left join fetch p.submissions s left join fetch s.result r where p.id = :participationId and (s.id = (select max(id) from p.submissions) or s.id = null)")
```

Comment:
not sure why you change this here, but this would mean the feedback is not included any more, right?

Reply:
This query was only used by the Resource method for fetching the build logs. The feedback was never used.